Misc/NEWS.d/next/Core and Builtins/2017-09-20-07-34-03.bpo-31530.adIJqt.rst

```diff
@@ -0,0 +1 @@
+Prevent crashes when multiple threads iterate of a file at once.
```

Comment:
Is "iterate of" correct?

Reply:
No, actually this file is supposed to be gone.